client/options.go

```diff
@@ -74,7 +75,7 @@ const (
 	defaultIdentifierPoolSize = 8192
 
 	// defaultWriteOpPoolSize is the default write op pool size
-	defaultWriteOpPoolSize = 1000000
+	defaultWriteOpPoolSize = 262144
```

Comment:
After looking at how many writes are open at any given time (i.e. num goroutines) we seem to peak at 60-80k, the original number was arbitrarily picked.